packages/@aws-cdk/integ-runner/README.md

```diff
@@ -0,0 +1,276 @@
+# integ-runner
+
+
+## Overview
+
+
+## Usage
+
+- Run all integration tests in `test` directory
+
+```bash
+integ-runner
+```
+
+This will look for all files that match the naming convention of `/integ.*.ts$/`. Each of these files will be expected
+to be a self contained CDK app. The runner will execute the following for each file (app):
+
+1. Check if a snapshot file exists (i.e. `/integ.*.expected.snapshot$/`)
+2. If the snapshot does not exist
+	2a. Synth the integ app which will produce the `integ.json` file
```

Comment:
If I understand it correctly, for each `integ.*.ts`, it will generate an `integ.json` file plus as many snapshots as there are tests (instances of `IntegTest`) in the `integ.*.ts` file. Is this correct?

Reply:
I was thinking that there would be a 1:1 between `integ.*.ts` and snapshots. Maybe we change the name of the construct to something like `IntegTestCase`?

An `integ.*.ts` file contains an "Integration Test" which corresponds to a single CDK application (i.e. one snapshot). Within that application we can create any number of `Stacks` which we then use in "test cases". 

We could also model it differently where you create a single `IntegTest` and then register any test cases to it `integTest.addTestCase()`